### ISSUE TITLE
Hyperparameters for ImageNet100 and ImageNette

### ISSUE THREAD

**Issue by AyushRoy2001** [jachansantiago/mode_guidance#1]
Dear authors,
We find your work very intuitive and the codebase is very clean. We were successfully able to implement and reproduce the reported results in the paper (+- 4 % accuracy).

However, we would greatly appreciate if you can provide us the fine-tuning hyperparameters of MinMax diffusion used for ImageNet100 and ImageNette to run the train_dit.py/finetune_dit.py (both are basically the same code)
